feature/src/main/java/com/terning/feature/mypage/MyPageRoute.kt

```diff
@@ -115,116 +130,218 @@ fun MyPageScreen(
     onQuitClick: () -> Unit,
     onNoticeClick: () -> Unit,
     onOpinionClick: () -> Unit,
+    onEditClick: () -> Unit,
     modifier: Modifier = Modifier,
     name: String = "",
-    authType: String = "",
+    profile: String = ""
 ) {
-    Box(
-        modifier = modifier.fillMaxSize(),
-        contentAlignment = Alignment.BottomCenter
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(Back)
     ) {
-        TerningImage(
-            modifier = Modifier.fillMaxSize(),
-            painter = R.drawable.ic_terning_mypage,
+        UserProfile(
+            name = name,
+            profile = profile,
+            onEditClick = { onEditClick() }
```

Comment:
이렇게 람다 형식으로 함수를 넘기는 이유가 있을까요??
람다가 새로운 객체를 생성하는 개념이라, 굳이 필요가 없는 경우엔 사용하지 않는게 좋을 것 같아요~
```suggestion
            onEditClick = onEditClick
```

Reply:
맞습니당!! 안 그래도 급하게 코드 짜느라 놓친 부분인 것 같네요ㅜㅜ 조언 감사합니당
한 번 더 수정하도록 할게요~!!